src/ansys/fluent/core/session.py

```diff
@@ -86,6 +87,21 @@ def build_from_fluent_connection(self, fluent_connection: FluentConnection):
         self._preferences = None
         self._solverworkflow = None
 
+    @property
+    def id(self) -> str:
+        """Return the session id."""
+        return self.fluent_connection._id
+
+    def start_journal(self, file_path: str):
+        """Executes tui command to start journal."""
+        warnings.warn("Use -> journal.start()", DeprecationWarning)
+        self.fluent_connection.journal.start(file_path)
+
+    def stop_journal(self):
+        """Executes tui command to stop journal."""
+        warnings.warn("Use -> journal.stop()", DeprecationWarning)
+        self.fluent_connection.journal.stop()
+
```

Comment:
@prmukherj 

- FluentConnection should not be managing all these things - it's not its job. 
- It's only at the session level where it's known what services are needed so we should instantiate them there.
- FluentConnection should expose what the session objects need to instantiate services.